packages/opentelemetry-node/test/instr-mongoose.test.js

```diff
@@ -43,7 +43,7 @@ const testFixtures = [
             // Ref: https://github.com/mongodb/node-mongodb-native/blob/a8370367f7470962a834ddf36f9a6c62621d6345/package.json#L118
             node: '>=16.20.1',
         },
-        // verbose: true,
+        verbose: true,
```

Comment:
Reviewer note: This could be excluded. I added this as a first step to trying to observe flaky test failures from this test case. I've seen it fail in local runs twice now.